engine/src/main/java/org/hibernate/validator/internal/util/ReflectionHelper.java

```diff
@@ -259,36 +260,27 @@ public static Object getValue(Method method, Object object) {
 	}
 
 	/**
-	 * Indicates if the type is considered indexable (ie is an {@code Iterable}, an array or a {@code Map}).
+	 * Indicates if the type is considered a collection of elements (ie is an <code>Iterable</code>, an array or
```

Comment:
We prefer to use `{@code ...}` over `<code>...</code>`. The reasoning is layed out in the contributor guide.